packages/dd-trace/src/id.js

```diff
@@ -182,6 +182,6 @@ function writeUInt32BE (buffer, value, offset) {
   buffer[0 + offset] = value & 255
 }
 
-module.exports = function createIdentifier (value, radix) {
+module.exports = Object.assign(function createIdentifier (value, radix) {
   return new Identifier(value, radix)
-}
+}, { Identifier })
```

Comment:
this isn't necessary but is a helpful export for some `instanceof` checks we do in `llmobs/tagger.js` - seems all tests still pass as-is, if we're ok to do this!

Reply:
Let's not expose this as we should replace the current implementation with `BigInt` at some point, and any exposed internals will make that more difficult.